Makefile

```diff
@@ -25,3 +27,9 @@ install:
 
 clean:
 	rm runc
+
+validate: vet
+	script/validate-gofmt
+	go vet ./...
+
+ci: validate test
```

Comment:
can this be localtest instead because I will already have it running in a container :)